rules.ini

```
@@ -274,6 +274,7 @@ NVIDIA_Texture_Tools = (?:^|/)nvtt(?:_64)?\.dll$
 NVIDIA_TXAA = (?:^|/)GFSDK_TXAA
 NWJS = (?:^|/)nw\.(?:dll|pak)$
 OculusXRPlugin = (?:^|/)OVRPlugin\.dll$
+ODIN = (?:^|/)(?:lib)?odin\.(?:dylib|dll|so)$
```

```suggestion
ODIN = (?:^|/)(?:lib)?odin\.(?:dll|so)$
```
looking for macos libs is not needed since it seems unlikely to me that there will ever be a macos only game published on steam (or at least in a near futur) (& all apps are detected without this part)

(don't forget to fix your test strings to take that change in account)